tests/components/uptime_kuma/test_config_flow.py

```diff
@@ -0,0 +1,106 @@
+"""Tests for the Uptime Kuma config flow."""
+import aiohttp
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.uptime_kuma.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+    CONTENT_TYPE_JSON,
+)
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+FIXTURE_USER_INPUT = {
+    CONF_HOST: "https://uptimekuma.test.com",
+    CONF_PORT: 443,
+    CONF_USERNAME: "user",
+    CONF_PASSWORD: "pass",
+    CONF_VERIFY_SSL: True,
+}
+
+
+async def test_show_authenticate_form(hass: HomeAssistant) -> None:
+    """Test that the setup form is served."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result["step_id"] == "user"
+
+
+async def test_connection_error(
+    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker
+) -> None:
```

Comment:
This is trying to use a socket, so I don't think `aioclient_mock` is working as you expect it to.